src/common/HaskellDo/Toolbar/View.hs

```diff
@@ -40,9 +40,12 @@ toolbar = rawHtml $ do
             li ! id "packageEditorButton" $ noHtml
             li ! id "toggleEditorButton" $ noHtml
             li ! id "toggleErrorButton" $ noHtml
+            li ! id "convertToPDFButton" $ noHtml
     packageEditorModal    -- Apparently, if we put this line
     openProjectModal      -- under this one. The open project modal doesn't work
     modalPromptPlaceholder "newDirectoryModal" "New Directory" "Choose a name for the new directory"
+    convertToPDFModal
+    convertToPDFModalFail
```

Comment:
Actually it's a mistake by my part, instead of making a separate component for each modal I just jammed em all into the toolbar, which is awful, see #76